db/migrate/20160701161129_remove_non_alphanumeric_chars_from_developer_username.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveNonAlphanumericCharsFromDeveloperUsername < ActiveRecord::Migration
+  def up
+  	Developer.find_each do |developer|
+	  if developer.username != developer.username.gsub(/[^0-9a-z ]/i, '')
```

Comment:
Same as above-- I think we don't want to allow spaces in the username, given how they are used in URL generation.